docs/reference/search/clear-scroll-api.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[clear-scroll-api]]
+=== Clear scroll API
+++++
+<titleabbrev>Clear scroll</titleabbrev>
+++++
+
+Clears the search context and results for a
+<<request-body-search-scroll,scrolling search>>.
+
+////
+[source,console]
+--------------------------------------------------
+GET /_search?scroll=1m
+{
+  "size": 1,
+  "query": {
+    "match_all": {}
+  }
+}
+--------------------------------------------------
+// TEST[setup:twitter]
+////
+
+[source,console]
+--------------------------------------------------
+DELETE /_search/scroll
+{
+    "scroll_id" : "DXF1ZXJ5QW5kRmV0Y2gBAAAAAAAAAD4WYm9laVYtZndUQlNsdDcwakFMNjU1QQ=="
+}
+--------------------------------------------------
+// TEST[continued]
+// TEST[s/DXF1ZXJ5QW5kRmV0Y2gBAAAAAAAAAD4WYm9laVYtZndUQlNsdDcwakFMNjU1QQ==/$body._scroll_id/]
+
+[[clear-scroll-api-request]]
+==== {api-request-title}
+
+`DELETE /_search/scroll/<scroll_id>`
+deprecated:[7.0.0]
+
+`DELETE /_search/scroll`
+
+[[clear-scroll-api-path-params]]
+==== {api-path-parms-title}
+
+`<scroll_id>`::
+deprecated:[7.0.0]
```

Comment:
@jrodewig  Can you please share how did you know that <scroll_id> is deprecated as a path param?

Reply:
Hi @mayya-sharipova. While the API doesn't return a deprecation warning, the JSON spec indicates that this path param is deprecated:
https://github.com/elastic/elasticsearch/blame/master/rest-api-spec/src/main/resources/rest-api-spec/api/clear_scroll.json#L28

If that's not the case, I can update these docs and fix the JSON spec too. Thanks!

Reply:
Thanks @jrodewig. I have confirmed with es clients team that we indeed deprecating scroll_id as a path param in DELETE API.